packages/color-block/tailwind.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    presets: [require('@frontify/fondue/tailwind')],
+    content: ['src/**/*.{ts,tsx}'],
+    corePlugins: {
+        preflight: false,
+    },
+    theme: {
+        extend: {
+            width: {
+                'list-color-types': 'calc(100% - 306px)',
+                'cards-card': 'calc(25% - 16px)',
+            },
```

Comment:
That's look not so great and we prefer to use the dynamic properties
'list-color-types' = should be responsive no hardcoded values
`cards-card` = it is just a grid of 3
<img width="567" alt="Screenshot 2022-07-28 at 12 59 57" src="https://user-images.githubusercontent.com/6846950/181489894-5ede5466-92b6-4ed9-891a-b8a45f444e86.png">